client/src/styles/_colors.scss

```diff
@@ -1,4 +1,6 @@
-$blue: #2185d0;
```

Comment:
rename this file into `variables`
and remove import this file from all files
also, don't forget to fix lint errors